website/content/en/docs/reference/configuration/_index.md

```diff
@@ -34,7 +34,7 @@ include      = ["/var/log/apache2/*.log"]    # supports globbing
 ignore_older = 86400                         # 1 day
 
 # Structure and parse via Timber's Remap Language
```

Comment:
```suggestion
# Structure and parse via Vector's Remap Language
```

I realize this is unrelated, but since you're in the area, it could be nice to update that too.